src/components/item/settings/ItemSettingProperty.tsx

```diff
@@ -0,0 +1,56 @@
+import { Stack, Switch, Typography } from '@mui/material';
+
+type Props = {
+  onClick: (checked: boolean) => void;
+  title: string;
+  checked: boolean;
+  valueText: string;
+  id?: string;
+  disabled?: boolean;
+  icon?: JSX.Element;
+};
+
+const ItemSettingProperty = ({
+  onClick,
+  title,
+  checked = false,
+  id,
+  disabled,
+  icon,
+  valueText, // eslint-disable-next-line arrow-body-style
```

Comment:
Is there a reason to explicitly disable the `arrow-body-style` instead of returning the stack directly (in arrow body style) ?